python/sf_salaries.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import os
+from pyspark import SparkConf, SparkContext
+from pyspark.sql import SQLContext
+
+conf = SparkConf()
+conf.setAppName("SF Salaries Example")
+
+# InsightEdge config
+if len(sys.argv) < 3:
+    spaceName = "insightedge-space"
+    lookupGroup = "insightedge"
+    lookupLocator = "127.0.0.1:4174"
+else:
+    spaceName = sys.argv[1]
+    lookupGroup = sys.argv[2]
+    lookupLocator = sys.argv[3]
+
+conf.set("spark.gigaspaces.space.name", spaceName)
+conf.set("spark.gigaspaces.space.lookup.group", lookupGroup)
+conf.set("spark.gigaspaces.space.lookup.locator", lookupLocator)
+
+sc = SparkContext(conf=conf)
+sqlContext = SQLContext(sc)
+
+# load SF salaries dataset from file
+jsonFilePath = os.path.join(os.environ["SPARK_HOME"], "data/sf_salaries_sample.json")
```

Comment:
I thought about that. There are already other datasets in /data, so I decided to put it there.